### PR TITLE
docs(changelog): update changelog

### PR DESCRIPTION
**Pull request by QZera**
Update changelog for v0.2.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 
+## **v0.2.0** - Jul 2nd 2024
+
+### 🚀 Enhancements
+
+- Add support for filtering via a text field
+
+### 🧹 Chores
+
+- Add container packaging
```
Comment (maninak) [inline CHANGELOG.md:11]:
```suggestion
- Add docker container packaging configuration
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Radicle Planning Boards Change Log
 
+## **v0.2.0** - Jul 2nd 2024
+
+### 🚀 Enhancements
+
+- Add support for filtering via a text field
```
Comment (maninak) [inline CHANGELOG.md:7]: We could and perhaps should say more about this and its subfeatures.

For example that it searches titles and ids, that it does partial string matching, that it highlights on the card exactly which part of it matched, the keyboard shortcut, the escape key to unfocus, and anything else I'm forgetting.

**Review by maninak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (QZera) [reply CHANGELOG.md]: Done!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+  - Uses partial string matching on the id, title, and labels of cards
+  - Filters the cards while keeping their column and order
+  - Visually highlights the matched part(s) on the cards
```
Comment (maninak) [inline CHANGELOG.md:10]:
```suggestion
  - Visually highlights on each card the matched substring(s)
```

Comment (maninak) [reply CHANGELOG.md:10]: Nit αλλά επειδή είναι σημαντικό feature και το proposed copy το κάνει market είπα να το γράψω.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+  - Features shortcuts for focusing and blurring the filterbox (`Ctrl/Cmd+F` and `Escape` respectively)
+  - Syncs the filterbox value with the URL for enhanced shareability
```
Comment (maninak) [inline CHANGELOG.md:12]:
```suggestion
  - filtering state synced with and driven by the URL means that any links you copy at any time and send to others will result in the same cards shown on their screen
```

kinda on the nose but whatever :P

**Review by maninak — APPROVED** [review]
Approved with proposed changes. Choose your destiny.

File: CHANGELOG.md (hunk 3 above)
Comment (QZera) [reply CHANGELOG.md]: Done!

File: CHANGELOG.md (hunk 4 above)
Comment (QZera) [reply CHANGELOG.md]: 👍 done

**Comment by QZera** [comment]
Updated changelog as suggested, updated date, updated version

**Review by maninak — APPROVED** [review]

**Pull request merged**